frontend/src/app/Catalog/CatalogView/CatalogView.tsx

```diff
@@ -85,8 +100,106 @@ const CatalogView = (props: CatalogViewProps) => {
 			setOpen(true);
 		}
 	}, [coursePreview, data]);
+	
+	
+	useEffect(() => {
+		if (course !== null) {	
+			if (false && legacyId !== null) {		
```

Comment:
This code is never executed becuase you inserted a false.

Reply:
removed